docs/cep/stream-workers-example.md

```diff
@@ -544,152 +530,268 @@ print("Samples", client.get_stream_app_samples())
+        return false;
+      } else {
+        console.log("Another issue has been detected.");
+        console.log(e);
+      }
+    }
+  }
+
+  async function hasQueryWorker (queryWorkerName) {
+    try {
+      const res = await client.getRestqls();
+
+      for (i = 0; i < res.result.length; i++) {
+        if (res.result[i].name === queryWorkerName) {
+          console.log(`Query worker ${queryName} already exist.`);
+          return true;
         }
+      }
+      console.log(`Query worker ${queryName} does not exist yet.`);
+      return false;
+    } catch (e) {
+      console.log(messageHandler(e));
     }
+  }
+
+  async function creatingQueryWorker () {
+    console.log(`--- Creating insert query worker "${insertDataValue.query.name}" if needed`);
+    if (!await hasQueryWorker(queryName)) {
+      client.createRestql(queryName, queryValue, parameter);
+      console.log(`Query worker ${queryName} has been created successfully.`);
+    }
+  }
+
+  async function creatingStreamApplication () {
+    let result = await client.get();
+
+    console.log("--- Validating stream application definition");
+    result = await client.validateStreamApp(appDefinition);
+    console.log(result);
+    // By default, the stream application is created in the local region. Optionally, you can send dclist to deploy stream
+    // app in all / selected regions
+    console.log("--- Creating stream application");
+    if (await hasStreamApp(streamAppName)) {
+      result = await client.getStreamApp(streamAppName);
+    } else {
+      result = await client.createStreamApp([], appDefinition);
+    }
+    console.log(result);
+
+    console.log("--- Getting stream application instance: " + streamAppName);
+    result = await client.getStreamApp(streamAppName);
+    console.log(result);
+  }
+
+  async function enablingStreamApplication () {
+    console.log("--- Enable stream application " + streamAppName);
+    // Enable app using change_state function
+    const result = await client.activateStreamApp(streamAppName, true);
+    console.log(result);
+    console.log("The connection has been opened.");
+  }
+
+  async function updatingSteamApplication () {
+    const app = await client.streamApp(streamAppName);
+
+    console.log("--- Updating stream application " + streamAppName);
+    await app.updateApplication([], updatedAppDefinition);
+    // Enable app using change_state function
+    await app.activateStreamApplication(true);
+  }
+
+  async function insertingData () {
+    console.log("Waiting for 30 seconds for all the resources to be ready");
+    await sleep(30000);
+    console.log("--- Inserting data to " + sourceCollectionName);
+    for (i = 1; i <= 50; i++) {
+      await client.executeRestql(queryName, { weight: i });
+      console.log(i);
+    }
+  }
+
+  async function disablingStreamApplication () {
+    await client.activateStreamApp(streamAppName, false);
+    console.log("The connection has been closed.");
```

Comment:
This doc is inconsistent with whether periods are used or not in the log. Please choose one and stick with it. Personally, I prefer the periods at the end of sentences, but I am not sure what is normal in JS logs.

Reply:
Done. I have actually removed the periods since the periods were mostly present in the logs added by us and the existing file logs didn't have periods it seems